cypress/support/pages/updateSettings.pageObject.cy.js

```diff
@@ -0,0 +1,59 @@
+import PageObject from '../PageObject';
+
+class UpdateSettingsPageObject extends PageObject {
+  url = '/settings';
+
+  get usernameField() {
+     return cy.getByDataCy('settings-username');
+  }
+
+  get bioField() {
+    return cy.getByDataCy('settings-bio');
+ }
+
+  get emailField() {
+  return cy.getByDataCy('settings-email');
+}
+
+  get newPasswordField() {
+    return cy.getByDataCy('settings-password');
+}
+
+  get updateSettingsBtn() {
+    return cy.getByDataCy('settings-submit');
+  }
+
+  get logOutBtn() {
+    return cy.getByDataCy('settings-logout-btn');
+  }
+
+  fillUsername(username) {
+    this.usernameField.clear().type(username);
+  }
+
+  fillBio(bio) {
+    this.bioField.type(bio);
```

Comment:
Consider clearing the bio field before typing to ensure no previous text interferes with the input.